maven-resolver-provider/pom.xml

```diff
@@ -103,6 +107,13 @@ under the License.
       <artifactId>failureaccess</artifactId>
       <optional>true</optional>
     </dependency>
+
+    <!-- Is must due PluginDescriptor as it uses Plexus stuff -->
+    <dependency>
+      <groupId>org.eclipse.sisu</groupId>
+      <artifactId>org.eclipse.sisu.plexus</artifactId>
```

Comment:
Needed for model and as builder throws PlexusConfEx :sob: 

Maybe just slurp in the plugin.xml as DOM?